tests/functional/vault/test_strategies.py

```diff
@@ -314,8 +354,42 @@ def test_ordering(gov, vault, TestStrategy, rando):
     with brownie.reverts():
         vault.addStrategyToQueue(removed_strategy, {"from": gov})
 
+def test_add_strategy_to_queue(gov, management, vault, TestStrategy, strategy, other_strategy, rando):
```

Comment:
nit: to be stylistically consistent with `test_addStrategy`, I guess this could be called `test_addStrategyToQueue`?